Source/OnlineSubsystemEpic/Private/OnlineUserInterfaceEpic.cpp

```diff
@@ -672,9 +680,11 @@ bool FOnlineUserEpic::GetAllUserInfo(int32 LocalUserNum, TArray< TSharedRef<clas
 
 						TSharedRef<FUserOnlineAccount> localUser = MakeShared<FUserOnlineAccountEpic>(epicNetId);
 						localUser->SetUserAttribute(USER_ATTR_COUNTRY, country);
+						localUser->SetUserAttribute(USER_ATTR_REALNAME, displayName);
```

Comment:
RealName := The actual real world name of a user
DisplayName := The name the user has chosen for himself
Alias := The name that was set for the queried user by the querying user

We should reflect that relationship.